site/_posts/2021-11-23-fusionauth-re-invent.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: blog-post
+title: "We'll see you at AWS re:Invent"
+description: FusionAuth is going to AWS re:Invent. Stop by and see us!
+author: Dan Moore
+image: blogs/fusionauth-aws-reinvent-2021/see-you-at-aws-reinvent-header-image.png
+category: blog
+excerpt_separator: "<!--more-->"
+---
+
+AWS re:Invent is one of the largest technical conferences. In the cloud space, it is the go-to event. In 2019, there were north of 50,000 attendees; this year the number of attendees is lower, but there'll still be a lot of folks in Las Vegas, NV the week after Thanksgiving. The FusionAuth team will be among them, and we'd love to see you.
```

Comment:
```suggestion
AWS re:Invent is one of the largest technical conferences in the United States. In the cloud space, it is the go-to event. This year the number of attendees is rapidly approaching 20,000; there'll be a lot of folks in Las Vegas, NV the week after Thanksgiving! The FusionAuth team will be among them, and we'd love to see you.
```

Reply:
Thanks, I tweaked it a bit, good feedback to not focus on the lower number.